src/python/pants/engine/internals/native.py

```diff
@@ -85,12 +138,15 @@ def generator_send(
                 )
             else:
                 raise ValueError(f"internal engine error: unrecognized coroutine result {res}")
-        except StopIteration as e:
```

Comment:
Might as well capitalize the i in "internal engine error" while you're in here.

Reply:
Done!